From: Santosh Ananthakrishnan <santosh@dropbox.com>
Date: Fri, 30 Mar 2018 07:02:13 +0000
Subject: xstrdup rule->program

---
 server/commands.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/server/commands.c b/server/commands.c
index cc87ea5..72c2858 100644
--- a/server/commands.c
+++ b/server/commands.c
@@ -217,7 +217,7 @@ create_argv_command(struct rule *rule, struct process *process,
         req_argv[1] = xstrdup("-u");
         req_argv[2] = xstrdup(rule->sudo_user);
         req_argv[3] = xstrdup("--");
-        req_argv[4] = rule->program;
+        req_argv[4] = xstrdup(rule->program);
         j = 5;
     } else {
         program = strrchr(rule->program, '/');
